stations.csv

```diff
@@ -4274,7 +4274,7 @@ id;name;slug;uic;uic8_sncf;latitude;longitude;parent_station_id;country;time_zon
 5642;Villefranche-Vernet-les-Bains;villefranche-vernet-les-bains;8778468;87784686;42.591998;2.370396;;FR;Europe/Paris;f;f;f;t;f;f;FRVIV;VVB;t;;f;8700454;f;;f;;f;;f;;f;;f;8700454;f;;f;;f;;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv5642;
 5643;Valréas Rond-Point;valreas-rond-point;8724008;87240085;44.387537;4.986586;;FR;Europe/Paris;f;f;f;t;f;f;FRVIW;;t;;f;8705829;f;;f;;f;;f;;f;;f;8705829;f;;f;;f;;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv5643;
 5644;Vix Centre;vix-centre;8748508;87485086;46.364625;-0.872816;;FR;Europe/Paris;f;f;f;t;f;f;FRVIX;;t;;f;8702450;f;;f;;f;;f;;f;;f;;f;;f;;f;;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv5644;
-5645;Nyons Centre;nyons-centre;8724014;87240143;44.360387;5.139134;;FR;Europe/Paris;f;f;f;t;f;f;FRVIY;;t;;f;8705830;f;;f;;f;;f;;f;;f;;f;;f;;f;;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;ニヨン;;;;Ньон;;;尼永;urn:trainline:public:nloc:csv5645;
+5645;Nyons Cave Cooperative;nyons-cave-cooperative;8724014;87240143;44.360387;5.139134;;FR;Europe/Paris;f;f;f;t;f;f;FRVIY;;t;;f;8705830;f;;f;;f;;f;;f;;f;;f;;f;;f;;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;ニヨン;;;;Ньон;;;尼永;urn:trainline:public:nloc:csv5645;
```

Comment:
The coordinates are not correct
https://www.google.com/maps/place/Nyons+Cave+Coop%C3%A9rative/@44.3603908,5.1365537,17z/data=!4m12!1m5!3m4!2zNDTCsDIxJzM3LjQiTiA1wrAwOCcyMC45IkU!8m2!3d44.360387!4d5.139134!3m5!1s0x12ca83e2acafd769:0x63c292cc436dd0e6!8m2!3d44.36005!4d5.13275!16s%2Fg%2F11sk62_rgw?entry=ttu&g_ep=EgoyMDI1MDIxOS4xIKXMDSoASAFQAw%3D%3D